src/pages/ViewNewsPaper/components/ShowContent.tsx

```diff
@@ -74,6 +76,25 @@ const ShowContent = ({ articleId, close }: ShowContentProps) => {
     }
   }
 
+  const updateArticleNote = async () => {
+    try {
+      const res = await axios.post(process.env.REACT_APP_API_URL + '/note/', {
+        text: note,
+        user_id: 1,
+        document_id: articleId
+      })
+
+      if (!res.data.success) {
+        console.log('error in adding note')
+        return
+      } else {
+        setUpdateNoteMsg('Thank you for your note.')
+      }
+    } catch (err) {
+      console.log(err)
```

Comment:
use `setUpdateNoteMsg` to show an error message (for example `An error has occurred`)

Reply:
If you could also add the error handling to the rest of the file that would be wonderful

Reply:
done, pls check